packages/paste-theme/src/themeProvider.tsx

```diff
@@ -52,7 +52,8 @@ const ThemeProvider: React.FunctionComponent<ThemeProviderProps> = ({
   ...props
 }) => {
   const prefersReducedMotion = useReducedMotion();
-  React.useMemo(() => {
+  // was causing error in storybook render.
+  React.useEffect(() => {
```

Comment:
question (blocking): what was the error? I had recently made this change as part of this PR https://github.com/twilio-labs/paste/pull/2328 so this change would introduce a bug

Reply:
Yes, I need to come back to this before the pr is ready for review. I unfortunately didn't get a screen capture of it, and I need to confirm it is still being reported.

I used `useEffect` here as a quick and dirty fix to get rid of the noise, not as a final fix. Thanks!